corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -320,9 +320,9 @@ <h4 class="panel-title accordion-toggle">
               {% if not forloop.first %}<br>{% endif %}
               <dt>{{ key }}</dt>
               {% if attachment.is_image %}
-                <dd><img src="{% url "form_attachment_view" domain=domain instance_id=instance.form_id attachment_id=key %}"></dd>
+                <dd><img src="{% url "api_form_attachment" domain=domain instance_id=instance.form_id attachment_id=key %}"></dd>
               {% else %}
-                <dd><a href="{% url "form_attachment_view" domain instance.form_id key %}" download="{{ key }}">{% trans "Download" %}</a></dd>
+                <dd><a href="{% url "api_form_attachment" domain instance.form_id key %}" download="{{ key }}">{% trans "Download" %}</a></dd>
```

Comment:
any reason not to update this to show the video control as well?

Reply:
you mean on the form data page? I can check with product if they want that change.

The view I added however will still stay because the project wants to show user just the recording and not the full form.